pkg/kbld/cmd/unprocessed_image_urls.go

```diff
@@ -5,14 +5,16 @@ package cmd
 
 import (
 	"sort"
+
+	"sigs.k8s.io/yaml"
 )
 
 type UnprocessedImageURL struct {
-	URL string
+	URL string `json:"image"`
 }
 
 type UnprocessedImageURLs struct {
-	urls map[UnprocessedImageURL]struct{}
+	urls map[UnprocessedImageURL]struct{} `json:"unresolved"`
```

Comment:
hmm, ytt doesnt have map key as top level when doing --files-inspect/--data-values-inspect. but its also probably a good idea to have it... should we rename this to `images`?